filament/src/RendererUtils.cpp

```diff
@@ -150,7 +151,17 @@ FrameGraphId<FrameGraphTexture> RendererUtils::colorPass(
                 data.depth = builder.read(data.depth, FrameGraphTexture::Usage::DEPTH_ATTACHMENT);
 
                 data.color = builder.write(data.color, FrameGraphTexture::Usage::COLOR_ATTACHMENT);
-                data.depth = builder.write(data.depth, FrameGraphTexture::Usage::DEPTH_ATTACHMENT);
+                if (engine.getConfig().stereoscopicType == StereoscopicType::MULTIVIEW) {
+                    // Add sampleable usage flag for depth in multiview rendering, othewise it's
+                    // treated renderbuffer in the backend and crashed.
+                    data.depth = builder.write(data.depth,
+                        FrameGraphTexture::Usage::DEPTH_ATTACHMENT |
+                        FrameGraphTexture::Usage::SAMPLEABLE);
+                    layerCount = engine.getConfig().stereoscopicEyeCount;
```

Comment:
I think here we need an `assert_invariant()` that data.depth and data.color have at least `layerCount` layers.

Also on line 115 `data.depth` should be created with `.depth` of `layerCount`. And `colorBufferDesc` must also have `layerCount` `.depth`.


I *think* if post-processing is disabled, this will work anyways because `data.depth` and `data.color` will be "magically" replaced by the "real" swapchain, which, in theory should be set-up properly.

I am okay with making this assumption for now, but we absolutely need a giant comment/TODO/FIXME here.

Reply:
Yeah as we said offline, the rest (setting the number of depth, etc) is in workspace. I'll soon make a following PR that handles what you said here.